packages/open-next/src/adapters/cache.ts

```diff
@@ -523,7 +508,7 @@ export default class S3Cache {
 
   private async deleteS3Objects(key: string) {
     try {
-      const regex = new RegExp(`\.(json|rsc|html|body|meta|fetch|redirect)$`);
+      const regex = new RegExp(`\.(fetch|cache)$`);
```

Comment:
I think this is handled in Derek's PR but the `\.` should have been `\\.` to escape the RegExp constructor. Since there's no dynamic values here, it might be better just to use the inline one, eg: `/\.(fetch|cache)$/`